test/Microsoft.IdentityModel.JsonWebTokens.Tests/JsonWebTokenHandler.ValidateTokenAsyncTests.Audience.cs

```diff
@@ -0,0 +1,213 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable enable
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.IdentityModel.TestUtils;
+using Microsoft.IdentityModel.Tokens;
+using Xunit;
+
+namespace Microsoft.IdentityModel.JsonWebTokens.Tests
+{
+    public partial class JsonWebTokenHandlerValidateTokenAsyncTests
+    {
+        [Theory, MemberData(nameof(ValidateTokenAsync_AudienceTestCases))]
```

Comment:
Please make sure 'DisableDiscoveryEnumeration = true' is added to all tests.

Reply:
We could add that as a follow-up to all the tests right? I think it's something we should fix all-up and maybe do in one large PR like @kellyyangsong did in SAL today?

Reply:
I'm working on it :)

Reply:
@jennyf19 that PR was committed, we should follow the pattern for all new work.

Reply:
Added this to the new tests.